duckduckgo/macos-browser
# Phishing Detection Privacy Dashboard Entrypoint

Task/Issue URL: https://app.asana.com/0/1199230911884351/1208149630394249/f
Tech Design URL: https://app.asana.com/0/481882893211075/1207220724600204/f
CC:

**Description**:
Implement AddressBarButtons to reflect designs from https://app.asana.com/0/0/1207896057014803/f

**Steps to test this PR**:
1. Visit privacy-test-pages.site/security/badware/phishing.html
2. Ensure error page is shown
3. Click through the warning
4. Red alert circle should be shown for the privacy dashboard entrypoint button



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
